ee/desktop/user/menu/menu_icon.go

```diff
@@ -53,6 +53,17 @@ func getIcon(icon menuIcon) []byte {
 			assets.MenubarCircleXMonochromeIco,
 			assets.MenubarCircleXMonochromePng,
 		)
+	case CircleDotIcon:
+		return chooseIcon(
+			assets.MenubarCircleDotDarkmodeIco,
+			assets.MenubarCircleDotDarkmodePng,
+			assets.MenubarCircleDotLightmodeIco,
+			assets.MenubarCircleDotLightmodePng,
+			assets.MenubarCircleDotLightmodeShadowIco,
+			assets.MenubarCircleDotLightmodeShadowPng,
+			assets.MenubarCircleDotMonochromeIco,
+			assets.MenubarCircleDotMonochromePng,
+		)
 	default:
```

Comment:
Can you make default better while you're here?

Reply:
https://github.com/kolide/launcher/pull/1548 🙂